COPYING

```diff
@@ -4,7 +4,7 @@ Source: http://github.com/ceph/ceph-nvmeof
 Files: *
 License: LGPL-3.0-or-later (see LICENSE)
 
-Files: proto/nvme_gw.proto
+Files: control/proto/gateway.proto
```

Comment:
I would keep the folder with protobuf definition at the highest level as it may be used to generate API externally.
For example, we will want to generate golang artifacts to use in Kubernetes for the CSI driver we will write